### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by GuptaEesh**
Cards contain:
>Types
   1. Simple Card
   2. Heading below image
   3. Text over image
   4. Without right footers
   5. Coming-Soon Cards
   6. Horizontal Cards
cards

### CODE REVIEW

File: components/cards.css [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  top: 1rem;
+}
+
+.card-three img {
+  filter: brightness(0.5);
+}
+
+.card-four .card-secondary-footer {
+  display: none;
+}
```
Comment (saadahmadhere) [inline components/cards.css:85]: add space between 2 class styles

**Comment by saadahmadhere** [comment]
Rest looks good.

**Review by MaruthiPothuganti — APPROVED** [review]
Looks good to merge👍

File: components/cards.css (hunk 1 above)
Comment (GuptaEesh) [reply components/cards.css:85]: done✅

**Pull request merged**